### ISSUE TITLE
Project Status: Is this still maintained?

### ISSUE THREAD

**Issue by pythops** [fede1024/rust-rdkafka#699]

**Comment by amineSt** [comment]
+1

**Comment by fede1024** [comment]
Hello, the project is currently maintained by a few DataDog employees, however the amount of time they can dedicate to this project is currently very limited. I might be finally able to go back to this project in the coming weeks and pick up some work.

If anyone who has already contributed extensively to the library wants to step up and become a maintainer, I'd be happy to discuss the option with them.

**Comment by pythops** [comment]
Good to know, thanks :)
I am open to contributing to the project,  but perhaps we could start with some triage first.

**Comment by rukai** [comment]
Hi, I have not contributed to this project before (but thats also currently impossible due to inactive maintainers)
But I have contributed to many related projects e.g. [the scylla driver](https://github.com/scylladb/scylla-rust-driver/pulls?q=is%3Apr+author%3Arukai+is%3Aclosed) and [the kafka-protocol crate](https://github.com/tychedelia/kafka-protocol-rs/pulls?q=author%3Arukai)

This is not a critical crate for us, but it is in our dep tree, so I would like to volunteer as a maintainer.
My responsibilities would be purely basic maintenance. Things like updating dependencies and resolving issues [like this one](https://github.com/fede1024/rust-rdkafka/pull/688). At the moment I have no interest in adding new features.

**Comment by Kinses38** [comment]
Any movement on this?
https://www.datadoghq.com/blog/engineering/streaming-platform-kafka-custom-abstractions/
I can see mention of no longer using this library here. 
Thanks

**Comment by j7nw4r** [comment]
I work at Microsoft on the EH team. I have bandwidth to help maintain this project. I'm not sure who is in charge of this at the moment, but if someone would make themselves known, I'd like to help.

**Comment by dennisrei** [comment]
My team relies on this project as well and I have time to help maintain this project as well. If more contributors are needed, I can help starting out with commenting on open issues and PRs

**Comment by fede1024** [comment]
@dennisrei yes please, that would be very welcome! :) starting out with open issues and PR review sounds great, thanks!
